lms/templates/banned.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+
+{% block page_content %}
+<div class="container">
+    <div id="login-container">
+        <div id="login" class="text-center">
+            <div id="course-logo" style="opacity: 0.6">
+                <img id="login-logo" src="{{ url_for('static', filename='avatar.jpg') }}" alt="תמונת הפרופיל של קורס פייתון" width="144" height="144">
+            </div>
+            <h1 id="main-title" class="h3 font-weight-normal">השהייה</h1>
+            <p>
+                המשתמש שלך מושהה עקב עבירה על תנאי הקורס.<br>

```

Comment:
```
המשתמש שלך הושעה על ידי מנהל המערכת.
לפרטים נוספים אנא פנה אל צוות הניהול.
```

Generally I like the idea with the mailto, but I don't want the system to be specific for our course.
One of my next tasks is to to language localization and to remove the logo of our course ^^"